com.vrcfury.vrcfury/Editor/VF/Service/SmoothingService.cs

```diff
@@ -72,10 +72,10 @@ public class SmoothingService {
 
         private Motion Smooth_(VFAFloat target, VFAFloat output, VFAFloat speedParam) {
             // Maintain tree - keeps the current value
-            var maintainTree = math.MakeMaintainer(output);
+            var maintainTree = math.MakeCopierForNegativeValues(output, output);
 
             // Target tree - uses the target (input) value
-            var targetTree = math.MakeCopier(target, output);
+            var targetTree = math.MakeCopierForNegativeValues(target, output);

```

Comment:
I'm concerned that this means that all `MakeZeroBasisFloat` usages may be broken after sitting in stations for similar reasons. Doesn't this mean that ANY dbt math, such as adding two values, would be broken because after sitting in the station, the existing value would become the parameter default and be added a second time?